specification/devtestlabs/resource-manager/Microsoft.DevTestLab/stable/2018-09-15/DTL.json

```diff
@@ -12846,23 +12683,6 @@
         }
       }
     },
-    "ServiceRunnerList": {
```

Comment:
Another Breaking change...

Reply:
This was never intended to be published. No customers should be using this.